man/tpm2_pcrallocate.1.md

```diff
@@ -0,0 +1,57 @@
+% tpm2_pcrallocate(1) tpm2-tools | General Commands Manual
+%
+% MARCH 2019
+
+# NAME
+
+**tpm2_pcrallocate**(1) - Change the allocated PCRs of a TPM
+
+# SYNOPSIS
+
+**tpm2_pcrallocate** [*OPTIONS*] [*ALLOCATION*]
+
+# DESCRIPTION
+
+**tpm2_pcrallocate**(1) - allows the user to specify an allocation for the TPM.
```

Comment:
It may be worth pointing out to the user that `tpm2_pcrallocate` is effective only after issuing a TPM reboot (or TPM_Startup(TPM_SU_CLEAR) if you want to be accurate).

Reply:
ACK

Reply:
Done